---
title: "'Finally'는 'Try' 문에서 한 번만 사용할 수 있습니다."
ms.date: 07/20/2015
f1_keywords:
- vbc30381
- bc30381
helpviewer_keywords:
- BC30381
ms.assetid: 4fa1d5fa-c54a-4f8c-9d66-9dbcc38c53bf
ms.openlocfilehash: 61328bd808e071670b72350c893960df78a96b66
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 06/04/2020
ms.locfileid: "84403747"
---
# <a name="finally-can-only-appear-once-in-a-try-statement"></a>'Finally'는 'Try' 문에서 한 번만 사용할 수 있습니다.
`Finally` 는 `Try...Catch...Finally` 블록을 완료하는 데 사용되므로 블록의 끝에 한 번만 나타날 수 있습니다.  
  
 **오류 ID:** BC30381  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
1. 중복 `Finally`를 찾아서 제거합니다.  
  
## <a name="see-also"></a>참고 항목

- [Try...Catch...Finally 명령문](../language-reference/statements/try-catch-finally-statement.md)
